### PR TITLE
[WIP] tutorials 1.x

### PR DESCRIPTION
**Pull request by frederikja163**
# Summary of the PR
Adds the source code of all tutorials to chapter one

# Related issues, Discord discussions, or proposals
On going discussions with PhilT and perksey on the discord

All credits goes to PhilT for the f# tutorials source code and bringing a bit of a functional philosophy into the tutorials.

### CODE REVIEW

File: Silk.NET.sln [hunk 1]
```diff
@@ -227,104 +227,6 @@ ProjectSection(SolutionItems) = preProject
 	examples\F#\OpenGL\Directory.Build.props = examples\F#\OpenGL\Directory.Build.props
 EndProjectSection
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Tutorial 1.1 - Hello Window", "examples\C#\OpenGL\Tutorial 1.1 - Hello Window\Tutorial 1.1 - Hello Window.csproj", "{13056B96-6B67-47E8-AFA5-3940C2D7F341}"
```
Comment (Perksey) [inline Silk.NET.sln]: Ensure you add the projects back

Comment (frederikja163) [reply Silk.NET.sln]: I will add them back once they have been made, but they werent even showing up before, something was corrupted or something... idk.

**Comment by Perksey** [comment]
No complaints so far :)

**Comment by Perksey** [comment]
Hi @PhilT. If you could check Discord at some point that'd be awesome :)

**Pull request closed without merging**